### PR TITLE
feat(gcp): Remove Classify and IgnoreError

### PR DESCRIPTION
**Pull request by yevgenypats**
Follow-up https://github.com/cloudquery/plugin-sdk/pull/60

---

Use the following steps to ensure your PR is ready to be reviewed

- [ ] Read the [contribution guidelines](../blob/main/CONTRIBUTING.md) 🧑‍🎓
- [ ] Test locally on your own infrastructure
- [ ] Run `go fmt` to format your code 🖊
- [ ] Lint your changes via `golangci-lint run` 🚨 (install golangci-lint [here](https://golangci-lint.run/usage/install/#local-installation)) 
- [ ] Update or add tests 🧪
- [ ] Ensure the status checks below are successful ✅

### CODE REVIEW

**Review by erezrokah — APPROVED** [review]

**Review by hermanschaaf — APPROVED** [review]

File: cli/examples/gcp.cq.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ spec:
   registry: grpc
   path: "localhost:7777"
   max_goroutines: 10000000
-  tables: ["*"]
+  tables: ["gcp_logging_metrics"]
```
Comment (hermanschaaf) [inline cli/examples/gcp.cq.yml:8]: was this change intentional?

**Comment by hermanschaaf** [comment]
Interesting error in the CLI tests, perhaps unrelated to the changes in this PR:

```
downloading 100% |█████████████████████████████| (3.7/3.7 MB, 284 MB/s)
    plugin_test.go:81: failed to start plugin /tmp/cq-plugins3909276279/plugins/github/yevgenypats/test/v1.0.1/cq-source-test_linux_amd64: fork/exec /tmp/cq-plugins3909276279/plugins/github/yevgenypats/test/v1.0.1/cq-source-test_linux_amd64: text file busy
--- FAIL: TestPluginManagerGetSourceClient (1.06s)
```

**Comment by hermanschaaf** [comment]
I think [this commit](https://github.com/cloudquery/cloudquery/pull/1757/commits/f658ecf092e4364e06b5a2c9b78de80cdba77a78) should fix the error mentioned above

File: cli/examples/gcp.cq.yml (hunk 1 above)
Comment (yevgenypats) [reply cli/examples/gcp.cq.yml:8]: Im going to remove those files (it's just my debug files) (put them in .gitignore)
